feeadjuster/feeadjuster.py

```diff
@@ -21,7 +28,35 @@ def get_ratio(our_percentage):
     return 50**(0.5 - our_percentage)
 
 
+def get_ratio_hard(our_percentage):
+    """
+    Return value is between 0 and 20: 0 -> 20; 0.5 -> 1; 1 -> 0
+    """
+    return 100**(0.5 - our_percentage) * (1 - our_percentage) * 2
+
+
+def get_chan_fees(plugin: Plugin, scid: str):
+    channels = plugin.rpc.listchannels(scid)["channels"]
+    for ch in channels:
+        if ch["source"] == plugin.our_node_id:
+            return { "base_fee_millisatoshi": ch["base_fee_millisatoshi"],
+                     "fee_per_millionth": ch["fee_per_millionth"] }
+
+
+def maybe_setchannelfee(plugin: Plugin, scid: str, base: int, ppm: int):
+    fees = get_chan_fees(plugin, scid)
+    if fees is None or base == fees["base_fee_millisatoshi"] and ppm == fees["fee_per_millionth"]:
```

Comment:
Could this ever be interpreted ambiguously?

`(A or B) and C` vs `A or (B and C)`

I guess the interpreter will always chose the right-hand version, but it'd be good to make this explicit.

Reply:
In general (all?) languages have `or` > `and` so that the boolean expressions/groups are separated by the `or`.